packages/browser/src/plugins/segmentio/index.ts

```diff
@@ -84,6 +85,10 @@ export function segmentio(
       ? batch(apiHost, deliveryStrategy.config)
       : standard(deliveryStrategy?.config as StandardDispatcherConfig)
 
+  const userAgentData = await clientHints(
```

Comment:
Thoughts on wrapping this in a try/catch?

You've got good error handling in `clientHints` already - my paranoia stems from the whole client hints API still being marked experimental, and safari/firefox haven't implemented it yet. My fear is they expose a partial implementation without `toJSON` (I think super unlikely scenario but a failure would mean the segment plugin fails to load which is pretty critical)